lib/dependency-manager-adapters/npm.js

```diff
@@ -43,7 +67,7 @@ module.exports = CoreObject.extend({
           name: dep,
           versionExpected: deps[dep],
           versionSeen: adapter._findCurrentVersionOf(dep),
-          packageManager: 'npm'
+          packageManager: adapter.configKey
```

Comment:
I didn't think that we would use an alternate config key in the `config/ember-try.js`, I assumed that we would continue to use `npm` as the key for each scenario...